### ISSUE TITLE
Implementation of truncate 

### ISSUE THREAD

**Issue by jydidier** [jydidier/keywordfs#1]
Here is one error message on truncate:
```
/home/didier/node_modules/keywordfs/keywordfs.js:460
        console.log('truncate', path, flags);
                                      ^

ReferenceError: flags is not defined
    at #truncate (/home/didier/node_modules/keywordfs/keywordfs.js:460:39)

Node.js v20.16.0
```

**Comment by jydidier** [comment]
Solved with v1.0.1

**Comment by jydidier** [comment]
Issue closed
